presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -487,6 +487,19 @@ Alter table operations are supported in the connector::
 
      ALTER TABLE iceberg.web.page_views DROP COLUMN location;
 
+When adding a column, we can indicate it as a partition column. To do that, we should indicate the ``transform`` functions for it,
```

Comment:
```suggestion
To add a new column as a partition column, identify the transform functions for the column.
```

"We" should be avoided in technical documentation. For some explanations of why, see the documentation recommended word list entries for 'we' in [GitLab](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#we), [Google](https://developers.google.com/style/word-list#we), and [Microsoft](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we).